### ISSUE TITLE
DOC: Simplify pandas theme footer

### ISSUE THREAD

**Issue by datapythonista** [pandas-dev/pandas#51536]
The sphinx theme that we're using didn't let us specify what exactly to show in our footer in the configuration file. So, what we did was to create a custom template to overwrite everything. Seems like now they [changed](https://github.com/pydata/pydata-sphinx-theme/pull/1184) this in their end, and we should be able to implement in our end the footer we want without requiring the extra template. I didn't check in detail, so some research is needed to see if this is really possible.

Their changes haven't been released yet, so we can't really get this into production until they do. But we can get the PR ready by temporary changing the pydata-sphinx-theme dependency to the git version.

**Comment by RamanathanVaradharajan** [comment]
take

**Comment by natmokval** [comment]
Hi @RamanathanVaradharajan, how is it going with this issue? 
If you don't mind I would like to work on it.

**Comment by datapythonista** [comment]
You can probably go ahead @natmokval, thanks for the help!

**Comment by natmokval** [comment]
Hi @datapythonista. I started to work on this issue and got stuck. I created a pr #52149 to show the changes I've made so far. Could you please help and advise me on what could be my next step?

**Comment by AbrhamSayd** [comment]
Hi how is it going with this issue?
I would like to continue.

**Comment by AbrhamSayd** [comment]
After researching and exploring different ways of displaying the pandas footer without using an extra template
I haven't found a solution to stop using it.
My closest approach was to use the 'html_context' setting to define the footer content inside the conf.py 
```
#Variable for the footer
footer = f"""{datetime.now().year}
pandas via <a href="https://numfocus.org">NumFOCUS, Inc.
</a> Hosted by <a href="https://www.ovhcloud.com">OVHcloud</a>"""

html_context = {
    "redirects": dict(moved_api_pages),
    "header": header,
    "footer": footer
}
```

And just pass the variable to the pandas_footer.html 

```
<p class="copyright">
    &copy {{footer}} 
</p>
```
But the template is still needed to render the context variable
```
html_theme_options = {
    "external_links": [],
    "footer_start": ["pandas_footer", "sphinx-version"],
    "github_url": "https://github.com/pandas-dev/pandas",
    "twitter_url": "https://twitter.com/pandas_dev",
    }
```
This is how its render
![image](https://github.com/pandas-dev/pandas/assets/25804457/287ee08d-2ed5-4170-96a5-dd8cd11727f5)

**Does anyone have any more ideas for getting rid of the template?**

**Comment by AbrhamSayd** [comment]
Hi @datapythonista  I've submitted a pull request that  proposes a potential solution.

**Comment by rsm-23** [comment]
@AbrhamSayd you should write `take` as a comment. It will assign the task to you.

**Comment by Spraveen8-chary** [comment]
Hi @datapythonista , I would like to work on it.

**Comment by SomaSahil** [comment]
@datapythonista Hi, I would like to work on this issue. Please let me know. thanks

**Comment by louisjh14** [comment]
take

**Comment by hriday-goyal** [comment]
take

**Comment by priyasingh814** [comment]
take

**Comment by maddiew95** [comment]
take

**Comment by maddiew95** [comment]
check out PR #61957 where I flat the footer. Hope this looks better.

<img width="1433" height="71" alt="Image" src="https://github.com/user-attachments/assets/51e8546e-ac19-4508-880f-a13031d1b2d5" />

**Comment by Prasad-JB** [comment]
Hi everyone 👋 I see this issue has had some great work done already but is still open. I’d be happy to help by either reviewing existing PRs or picking up where the last one left off. Could someone confirm the current status and whether contributions are still welcome here?

**Comment by bclaire92** [comment]
take

**Comment by Prasad-JB** [comment]
Hi everyone 👋,

I see there has been some great progress on simplifying the pandas theme footer, but the issue is still open. Could someone confirm the current status of this issue? Are contributions still welcome? I’d be happy to help by reviewing existing PRs or continuing from where the last one left off.

Thanks!

**Comment by jill-augustine** [comment]
take

**Comment by jill-augustine** [comment]
Hi @datapythonista, I researched this topic a bit more and there doesn't seem to be a way to remove the template doc completely. The only could suggest would be to replace `&copy {{copyright}} ` from `pandas_footer.html` with the new built-in
"copyright" option.
This would still leave us needed the pandas_footer.html to include the `via <a href="https://numfocus.org">NumFOCUS, Inc.</a> Hosted by <a href="https://www.ovhcloud.com">OVHcloud</a>.` which isn't much cleaner imo. 

I would suggest to close the issue because pydata-sphinx-theme doesn't currently allow HTML strings in the `conf.py` to be rendered correctly.

**Comment by jill-augustine** [comment]
`take`

**Comment by mroeschke** [comment]
> I would suggest to close the issue because pydata-sphinx-theme doesn't currently allow HTML strings in the conf.py to be rendered correctly.

Agreed, based on other PRs that have been close it doesn't appear like a net simplification so closing
